vercipe_frontend/frontend/src/app/create-board/create-board.component.html

```diff
@@ -1,22 +1,32 @@
 <mat-card id="createBoard">
   <mat-card-content>
     <mat-form-field>
-      <input matInput placeholder="title" [(ngModel)]="title"/>
+      <input matInput placeholder="title" (ngModel)="title">
     </mat-form-field>
 
+    <h3>Time:</h3>
+    <div id="time">
+      <mat-form-field>
+        <input matInput placeholder="prep time" (ngModel)="prepTime">
+      </mat-form-field>
+      <mat-form-field>
+        <input matInput placeholder="cook time" (ngModel)="cookTime">
+      </mat-form-field>
+    </div>
+
     <h3>Materials:</h3>
     <div *ngFor="let mat of materials; let index=index;">
       <mat-form-field>
-        <input matInput placeholder="item {{index+1}}" [(ngModel)]="materials[index]"
-         (keyup.enter)="addMat()"/>
+        <input matInput placeholder="item {{index+1}}" (ngModel)="materials[index]"
+         (keydown.enter)="addMat()">
```

Comment:
When a new input filed is added, we need to make the type focus to that part rather than stay in the same one